packages/opentelemetry-types/src/trace/trace_options.ts

```diff
@@ -14,6 +14,13 @@
  * limitations under the License.
  */
 
-export interface TraceOptions {
-  // TODO
-}
+type TraceOptionsSampled = 0x1;
```

Comment:
I should probably have had it in my initial example, but should this be exported as well?

Reply:
Why is this needed at all? Can't the constants be used directly instead? It would be cleaner to not have to repeat the underlying values.

Reply:
Well, the trouble here I think is that these types will just compile down to nothing in the eventual JS. In contrast, the `export const` will end up as something we can have regular JS import.

I agree this is an annoying edge case. Would be nice if we could do something like `export type TraceOptions = TRACE_OPTION_SAMPLED|TRACE_OPTION_UNSAMPLED;`, but that won't compile (see this [TS playground link](https://www.typescriptlang.org/play/#code/MYewdgzgLgBAygQQLIAUAyBRAIjAvDARgG4AoUSWAVQDlFVMd8AmUqATwAcBTGAFQCcAhsC4B5DlACW4CHnjJ02AD406irESA) where the error is "SAMPLED refers to a value but it is being used as a type here).

We could have less verbosity by having `export type TraceOptions = 0x1|0x2;`. and I'm fine with that. Would you prefer that?

Reply:
I would say yes since the meaning of the values can be understood by looking at the constants.

Reply:
Pushed the required change. PTAL